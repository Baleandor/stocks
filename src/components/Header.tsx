export default function Header() {
  return <h1 className="bg-none text-center text-4xl pt-3">Stock Market Dashboard</h1>;
}
