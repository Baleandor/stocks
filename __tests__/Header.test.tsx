import { render, screen } from "@testing-library/react";
import Header from "~/components/Header";

describe("Header component", () => {
  it("renders the header text", () => {
    render(<Header />); // ARRANGE

    const headerElement = screen.getByText("Stock Market Dashboard"); // ACT

    expect(headerElement).toBeInTheDocument(); // ASSERT

    expect(headerElement).toHaveClass("bg-none text-center text-4xl pt-3");
  });

  it("matches snapshot", () => {
    const { asFragment } = render(<Header />); // ARRANGE

    expect(asFragment()).toMatchSnapshot(); // ASSERT
  });
});
