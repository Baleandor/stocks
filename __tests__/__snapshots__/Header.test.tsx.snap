// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component matches snapshot 1`] = `
<DocumentFragment>
  <h1
    class="bg-none text-center text-4xl pt-3"
  >
    Stock Market Dashboard
  </h1>
</DocumentFragment>
`;
